email_account_validity/_base.py

```diff
@@ -47,10 +51,15 @@ def __init__(self, config: Any, api: ModuleApi, store: EmailAccountValidityStore
         try:
             app_name = self._api.email_app_name
             self._renew_email_subject = renew_email_subject % {"app": app_name}
-        except Exception:
+        except (KeyError, TypeError):
             # If substitution failed, fall back to the bare strings.
             self._renew_email_subject = renew_email_subject
 
+        (self._template_html, self._template_text,) = api.read_templates(
+            ["notice_expiry.html", "notice_expiry.txt"],
+            os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates"),
```

Comment:
Will be fixed by https://github.com/matrix-org/synapse/pull/10596, since part of its changeset is adding the new generic custom template directory in the list of directories the module API will look for custom templates in.